README.md

```diff
@@ -1,11 +1,11 @@
 # node-problem-detector
 
-[![Build Status](https://travis-ci.org/kubernetes/node-problem-detector.svg?branch=master)](https://travis-ci.org/kubernetes/node-problem-detector)  [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes/node-problem-detector)](https://goreportcard.com/report/github.com/kubernetes/node-problem-detector)
+[![Build Status](https://travis-ci.or,-detector.svg?branch=master)](https://travis-ci.org/kubernetes/node-problem-detector)  [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes/node-problem-detector)](https://goreportcard.com/report/github.com/kubernetes/node-problem-detector)
```

Comment:
Is this a typo?

Reply:
eep, that's definitely keyboard/trackpad selection damage.

Reply:
(It's also evidence that I'm a human... -- I'm actually fixing similar damage in someone else's repository...)